Patches/PlayerControlPatch.cs

```diff
@@ -393,7 +393,7 @@ public static bool Prefix(PlayerControl __instance, [HarmonyArgument(0)] PlayerC
                 }
             }
 
-            if (!target.IsProtected() && !Doppelganger.CheckDoppelVictim(target.PlayerId) && !Camouflage.ResetSkinAfterDeathPlayers.Contains(target.PlayerId))
+            if (!target.IsProtected() && !Doppelganger.CheckDoppelVictim(target.PlayerId) && !Main.OvverideOutfit.ContainsKey(target.PlayerId) && !Camouflage.ResetSkinAfterDeathPlayers.Contains(target.PlayerId))
```

Comment:
`!Doppelganger.CheckDoppelVictim(target.PlayerId)` Already useless because it already checks `Main.OvverideOutfit`